app/scripts/lib/setupSentry.js

```diff
@@ -666,9 +666,16 @@ export default function setupSentry({ release, getState }) {
   const startSession = async () => {
     const hub = Sentry.getCurrentHub?.();
     const options = hub.getClient?.().getOptions?.() ?? {};
-    if (hub && (await getSentryEnabled()) === true) {
+    const isMetaMetricsEnabled = (await getMetaMetricsEnabled()) === true;
```

Comment:
Why do we not want to use `getSentryEnabled` since it also handles MMI which always want metrics enabled?

Reply:
This is a typo from my side. sorry about that ! fixed in [1b9a62](https://github.com/MetaMask/metamask-extension/pull/25990/commits/1b9a62b2bc3cee88dbe709a49739805082a11fc8)